sarif/sarif_file_utils.py

```diff
@@ -23,7 +23,9 @@ def combine_code_and_description(code: str, description: str) -> str:
         code = code.strip()
         length_budget -= len(code) + 1
     # Allow extra space when truncating for continuation characters
-    length_budget_pre_continuation = length_budget - 20
+    length_budget_pre_continuation = length_budget - 5
+    if length_budget_pre_continuation < 10:
```

Comment:
What is the magic number 10? Do we somehow know that it's impossible for `description` to be less than 10 characters, and therefore don't bother even looking at `description` in this case?

Reply:
It's just an approximation of the minimum length of a description to be useful, in the strange circumstances of a very long issue code.  I've added a comment to clarify.